addons/website_sale/views/snippets.xml

```diff
@@ -6,18 +6,15 @@
         <t t-set="_classes" t-value="'s_wsale_products_searchbar_input mb-3'"/>
     </t>
 </template>
+
 <template id="s_products_searchbar" name="Products Search">
-    <section class="s_wsale_products_searchbar bg-200 pt24 pb24">
+    <section class="s_wsale_products_searchbar bg-200 pt48 pb48">
         <div class="container">
             <div class="row">
                 <div class="col-lg-8 offset-lg-2">
-                    <div class="card">
-                        <div class="card-body">
-                            <h2>Search for a product</h2>
-                            <p>We have amazing products in our shop, check them now !</p>
-                            <t t-call="website_sale.s_products_searchbar_input"/>
-                        </div>
-                    </div>
```

Comment:
@Cocographique Could you create a commit in the first stable branch where the bug occurs for this part? You can leave this one here, it will go away with forward-port. And remove the uppercase letter in the commit message (in your new PR, not here).

Reply:
@qsm-odoo Do you want me to create a specific task too?
[PR](https://github.com/odoo/odoo/pull/46829)

Reply:
Yes create a task (or multiple if meaningful) for the commits I did not merge and add the PR link in there (and remove the spec from the original task and add it in the new task(s)). And move it to RD Dev - Green if finished.